services/order.py

```diff
@@ -0,0 +1,30 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order
+from db.models import Ticket
+from db.models import User

```

Comment:
Import these models in one line, do not duplicate `from db.models` for several times